fv3core/external/gt4py

```diff
@@ -0,0 +1 @@
+../../external/gt4py
```

Comment:
Is this one of the symlinks we have discussed?

Reply:
I'm not positive I know what you mean, but i think yes?